be/src/exec/pipeline/olap_chunk_source.h

```diff
@@ -74,6 +74,7 @@ class OlapChunkSource final : public ChunkSource {
     vectorized::TabletReaderParams _params = {};
 
     vectorized::OlapScanNode* _scan_node;
+    TupleDescriptor* _tuple_desc = nullptr;
```

Comment:
need keep it as a field, you could directly get table name.

Reply:
ok

Reply:
TupleDescriptor need tuple_id which is not saved as field.

Reply:
Ok